flag.go

```diff
@@ -390,14 +392,30 @@ func (f *FlagSet) getFlagType(name string, ftype string, convFunc func(sval stri
 		return nil, err
 	}
 
-	sval := flag.Value.String()
+	envVal, exist := getFlagTypeFromEnv(flag.EnvVars)
+	var sval string
+	if exist {
+		sval = envVal
+	} else {
+		sval = flag.Value.String()
+	}
```

Comment:
When the both the flag value and the env var are specified, the flag value should be prioritized over the env value

https://github.com/containerd/nerdctl/issues/495